src/github.ts

```diff
@@ -160,13 +160,15 @@ async function getPiperBinaryNameFromInputs (stepName: string, version?: string)
   return piper
 }
 
-async function getTag (version?: string): Promise<string> {
+async function getTag (forAPI: boolean, version?: string): Promise<string> {
   let tag
   if (version !== undefined) {
     version = version.toLowerCase()
   }
   if (version === undefined || version === '' || version === 'master' || version === 'latest') {
     tag = 'latest'
+  } else if (forAPI) {
```

Comment:
maybe name it something like "innerSource"? "forAPI" is not so obvious

Reply:
but if tomorrow we will (for some reason) use API call for downloading open-source Piper or something else, then `innerSource` is not a good name. 
This function is about returning different "tags" (or let's say part of URL) depending on whether it is an API call or not, it is not about inner or open source.
How about `forAPICall`?